support-e2e/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Presume this goes with the type checking?

Reply:
Yeah exactly, I needed a basic tsconfig to do this 👍